src/ape/api/accounts.py

```diff
@@ -214,6 +316,13 @@ def __contains__(self, address: AddressType) -> bool:
             return False
 
     def _verify_account_type(self, account):
+        """
+        Check if the account is of the correct type.
+        Raise ``AccountError`` if the account in the contaier is not correct
```

Comment:
```suggestion
        Raise ``AccountError`` if the account in the container is not correct.
```

Reply:
Not needed. We removed the docstring since it is a protected method